From 30cba352e364894c0829affe665aab805202e5c4 Mon Sep 17 00:00:00 2001
From: BambooIV <ningdisheng@gmail.com>
Date: Tue, 15 Mar 2016 11:42:39 +0800
Subject: [PATCH 02/36] nx503a: ARM: dts: msm: Update device tree node for
 msm-pcm-loopback

cherry-pick from 4ef7b074451bb7b565a0583975f542d3d73c3d57

Update the name and properties for msm-pcm-loopback. Use qti instead
of qcom for device node and its properties.
---
 arch/arm/boot/dts/NX503A/msm8974.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/NX503A/msm8974.dtsi b/arch/arm/boot/dts/NX503A/msm8974.dtsi
index 382f6bd..785887c 100644
--- a/arch/arm/boot/dts/NX503A/msm8974.dtsi
+++ b/arch/arm/boot/dts/NX503A/msm8974.dtsi
@@ -1788,8 +1788,8 @@
 		compatible = "qcom,msm-lsm-client";
 	};
 
-	qcom,msm-pcm-loopback {
-		compatible = "qcom,msm-pcm-loopback";
+	qti,msm-pcm-loopback {
+		compatible = "qti,msm-pcm-loopback";
 	};
 
 	qcom,msm-dai-q6 {
-- 
1.9.1

